packages/notifier/src/publish-kit.js

```diff
@@ -12,117 +11,64 @@ const { Fail, quote: q } = assert;
 
 const sink = () => {};
 const makeQuietRejection = reason => {
-  const rejection = HandledPromise.reject(reason);
+  const { promise: rejection, reject } = makePromiseKit();
   void E.when(rejection, sink, sink);
-  return harden(rejection);
+  reject(reason);
+  return rejection;
 };
```

Comment:
What motivates this change?

Reply:
I am trying to contain the spread of the HandledPromise API, at least until we make more progress on the eventual-send proposal, which will almost certainly involve shimming a different API.

Reply:
For a promise that's born rejected, what's the value added by either of `HandledPromise` or `promise-kit`? Why not simply do
```js
const rejection = Promise.reject(reason);
```
?

Reply:
Good point.  `Promise.reject` it is.